src/main/java/seedu/duke/exceptions/ExceptionType.java

```diff
@@ -36,14 +36,17 @@
     EXTRA_PARAMETERS("Hmm, this function should not be called with parameters."),
     INVALID_INDEX("Sorry, that isn't the index of any command in the list."),
     SAME_DESCRIPTION("No change needed! You already have that description for your favourite command."),
-    EMPTY_INDEX("Oh no! I cannot detect the input index.");
+    EMPTY_INDEX("Oh no! I cannot detect the input index."),
+    LIST_INDEX_CORRUPTED("Oh no! it seems that this command has been corrupted.\n"
+            + "Don't worry, I have deleted it from your favourites list!"),
+    POSSIBLE_LOCS("Possible Locs shown");
```

Comment:
Why is this an exception?

Reply:
A dummy exception needs to be thrown to prevent an extra output on top of the similarloc output.
At the same time the dummy exception will be caught by execfav and auto clean the fauvist if a command is corrupted due to spell check

Reply:
Gotcha! So maybe you can merge first, then I will make the required changes to /route command as well!